line-bot-spring-boot/src/main/java/com/linecorp/bot/spring/boot/annotation/EventMapping.java

```diff
@@ -43,4 +43,6 @@
      * Priority of this mapping. Bigger mapping is preferentially searched and matched.
      */
     int priority() default DEFAULT_PRIORITY_VALUE;
+
+    String secretKey() default "";
```

Comment:
secretkey is not unique identifier of LINE BOT. (can be changed)
Maybe we define new name or using ChannelID.

![ece98aabc806e54ade8fff431cd0943a](https://cloud.githubusercontent.com/assets/1370068/25158474/9c5a0db0-24e3-11e7-8d32-48c3cd4becf3.png)